src/components/backend-ai-webui.ts

```diff
@@ -1452,24 +1440,17 @@ export default class BackendAIWebUI extends connect(store)(LitElement) {
                   </mwc-list-item>
                 `
               : html``}
-            ${!this.blockedMenuItem.includes('session')
-              ? html`
-                  <!-- <mwc-list-item graphic="icon" ?selected="${this._page ===
-                  'session'}" @click="${() =>
-                    this._moveTo(
-                      '/session',
-                    )}" ?disabled="${this.inactiveMenuItem.includes(
-                    'session',
-                  )}">
-                    <i class="fas fa-list-alt" slot="graphic" id="sessions-menu-icon"></i>
-                    <span class="full-menu">${_t(
-                    'webui.menu.Sessions',
-                  )} new</span>
-                  </mwc-list-item> -->
-                `
-              : html``}
-            ${this.supportServing && !this.blockedMenuItem.includes('serving')
-              ? html`
+            <!--<mwc-list-item graphic="icon" ?selected="${this._page ===
+            'session'}" @click="${() =>
+              this._moveTo(
+                '/session',
+              )}" ?disabled="${this.inactiveMenuItem.includes('session')}">
+              <i class="fas fa-list-alt" slot="graphic" id="sessions-menu-icon"></i>
+              <span class="full-menu">${_t('webui.menu.Sessions')} new</span>
+            </mwc-list-item>-->
+            ${!this.supportServing || this.blockedMenuItem.includes('serving')
```

Comment:
I also set the visibility of serving page not only dependent to Core version supports but also checking whether `serving` menu is included to blocklist.